### PR TITLE
Release eds-core-react@0.16.1

### LINKED ISSUE #1775: Release eds-core-react@0.16.1
hotfix ramda security warning

### PR DESCRIPTION
**Pull request by oddvernes**
Resolves #1775

### CODE REVIEW

File: libraries/core-react/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.16.1] - 2021-12-07
+
+### Changed
+
+- Bundle tree-shaked ramda instead of having it as dependency to avoid security warning for users ([#1768](https://github.com/equinor/design-system/issues/1768))
```
Comment (mimarz) [inline libraries/core-react/CHANGELOG.md:12]: maybe add; security warning for users by unused ramda functions? 😅

Comment (mimarz) [reply libraries/core-react/CHANGELOG.md:12]: just to clarify it was never really used, even though snyk flags the package as a whole...

**Review by mimarz — APPROVED** [review]
Maybe a small clarification, else it looks good 👍

File: libraries/core-react/CHANGELOG.md (hunk 1 above)
Comment (oddvernes) [reply libraries/core-react/CHANGELOG.md]: done😅

**Pull request merged**